.github/workflows/mvn-verify-check.yml

```diff
@@ -78,6 +89,32 @@ jobs:
           -Dmaven.javadoc.skip
           -Dmaven.scalastyle.skip=true
           -Drat.skip=true
+          
+  package-snapshotsOnly-aggregator:
+    needs: get-noSnapshot-versions-from-dist
+    strategy:
+      matrix: ${{ fromJSON(needs.get-noSnapshot-versions-from-dist.outputs.sparkSnapshotOnlyVersions) }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3 # refs/pull/:prNumber/merge
+
+      - name: Setup Java and Maven Env
+        continue-on-error: true
```

Comment:
This doesn't seem to work as intended. If one build fails, then it aborts all other builds in this job.